source/client-side-encryption/client-side-encryption.rst

```diff
@@ -630,7 +630,7 @@ redirect to /dev/null). Users can pass the argument --logpath to
 extraOptions.mongocryptdSpawnArgs if they need to inspect mongocryptd
 logs.
 
-Upon construction, the ClientEncryption MUST create a MongoClient to
+Upon construction, the MongoClient MUST create a MongoClient to
```

Comment:
I believe this is a typo, but correct me if I'm wrong.

Reply:
Oh good catch! ClientEncryption does not do automatic encryption, so it does not need to connect to or spawn mongocryptd.